spec/support/shared_examples/curriculum/rateable_examples.rb

```diff
@@ -55,6 +55,12 @@
       post send(comment_path, rating_id: rating_id, comment: 'This is a test')
       expect(response).to have_http_status(:ok)
     end
+
+    it 'allows comments with double quotes to be escaped and submitted' do
+      stub_a_valid_request({ data: { update_rating: {} } }.to_json)
+      post send(comment_path, rating_id: rating_id, comment: '"<body style="background-color:powderblue;">')
+      expect(response).to have_http_status(:ok)
```

Comment:
might be just a little more robust to also do: `expect(Rating.last.comment).to eq('\"<body style=\"background-color:powederblue;\">')` since I'm not 100% confident about how well the api is handling errors at the moment, and I worry that you'll get an :ok status regardless

Reply:
resolved thank you

Reply:
> might be just a little more robust to also do: `expect(Rating.last.comment).to eq('\"<body style=\"background-color:powederblue;\">')` since I'm not 100% confident about how well the api is handling errors at the moment, and I worry that you'll get an :ok status regardless

resolved thank you 😊

Reply:
sorry my suggested code had a typo in it `background-color:powederblue` should be `background-color:powderblue`, if you change that the test might pass?

Reply:
I am not sure the class Rating is exists in TC as it only captures the comment but not store it. However the expected change registers in the curriculum database.

Reply:
ah yes! I was forgetting this was TC